packages/graphql-toolbox/src/modules/EditorView/Editor.tsx

```diff
@@ -49,17 +49,18 @@ export interface Props {
 }
 
 export const Editor = ({ schema }: Props) => {
+    const theme = useContext(ThemeContext);
     const store = useStore();
     const settings = useContext(SettingsContext);
-    const theme = useContext(ThemeContext);
     const [loading, setLoading] = useState<boolean>(false);
     const [showDocs, setShowDocs] = useState<boolean>(false);
-    const refForQueryEditorMirror = useRef<EditorFromTextArea | null>(null);
+    const elementRef = useRef<HTMLDivElement | null>(null);
```

Comment:
Can this also be moved to the `GraphQLQueryEditor` file? (In the JSONEditor file the ref is in the same file as the editor)